lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -475,6 +513,50 @@ ExhaustiveSynthesis::synthesize(SMTLIBSolver *SMTSolver,
     llvm::errs() << "There are " << Guesses.size() << " Guesses\n";
 
 
+  std::map<Inst *, Inst *> InstCache;
+  std::map<Block *, Block *> BlockCache;
+
+  if (UseAlive) {
+    std::set<const Inst *> Visited;
+    if (exceeds64Bits(LHS, Visited))
+      llvm::report_fatal_error("LHS exceeds 64 bits");
+
+    Inst *Ante = IC.getConst(APInt(1, true));
+    for (auto PC : PCs ) {
+      Inst *Eq = IC.getInst(Inst::Eq, 1, {PC.LHS, PC.RHS});
+      Ante = IC.getInst(Inst::And, 1, {Ante, Eq});
+    }
+
+    AliveDriver Verifier(LHS, Ante);
+    for (auto &&G : Guesses) {
```

Comment:
this can wait until after merging but please work on a PR that adds debug prints in this loop that are equivalent (in terms of information content and in what gets printed at what debug level) to the existing non-alive guessing loop